src/main/resources/sessionLinksRecoveryEmailTemplate-feedbackSessionAccessLinks.html

```diff
@@ -0,0 +1,27 @@
+<p>Hello ${userName},</p>
+
+<p>
+    You are receiving this email because someone requested <a href="${teammateHomePageLink}">TEAMMATES</a> (an online peer evaluation system) to send access links to this email address. If you did not make that request, please ignore this email.
+</p>
+
+<p>
+    Below are the open/closed/published feedback sessions that are associated to the email ${recoveryEmail} within the past 180 days:
```

Comment:
`the open/closed/published feedback sessions` -> `the sessions`?

In any case, a session can be both open and published as they are two independent attributes.

Reply:
`associated to` -> `associated with`?